pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -377,9 +377,9 @@ func newInboundPassthroughFilterChains(env *model.Environment, node *model.Proxy
 
 		matchingIP := ""
 		if clusterName == util.InboundPassthroughClusterIpv4 {
-			matchingIP = util.InboundPassthroughBindIpv4
```

Comment:
??

Reply:
It is an awkward mistake... 
1. the tcp pass through listener is supposed to catch all the unknown traffic so it is listening 0.0.0.0/0  with any port.
2. The the 127.0.0.6 is bind to the connection from envoy to app e.g. 
```
envoy src address 127.0.0.6:<dynamic port> 
to
app service address such as 0.0.0.0:80 
```